libs/SalesforceSDKCommon/SalesforceSDKCommon/Classes/Logging/Lumberjack/DDASLLogger.h

```diff
@@ -0,0 +1,41 @@
+#import <Foundation/Foundation.h>
+#import <asl.h>
+
```

Comment:
```CocoaLumberjack``` begins here. We should consider moving it to a submodule like ```FMDB```, but that's for another pull request.